posthog/hogql/filters.py

```diff
@@ -111,4 +131,40 @@ def visit_placeholder(self, node):
             if len(exprs) == 1:
                 return exprs[0]
             return ast.And(exprs=exprs)
+        if node.field == "filters.dateRange.dateFrom":
```

Comment:
Hmm, maybe for brevity this could be `["filters", "dateRange", "from"]`? Although the actual query schema does not practice brevity